.github/CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# Owners of repository
+/.github/CODEOWNERS @calcof
+/README.md @calcof
+
+# Owners for introduction docs
+/docs/introduction/ @calcof @heoelri
```

Comment:
not that I want to get rid of Heyko, nor do I want to see my name there. But...why him on the intro? :D